autoware_planning_msgs/package.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>autoware_planning_msgs</name>
+  <version>0.0.0</version>
+  <description>Autoware planning messages package.</description>
+  <maintainer email="isamu.takagi@tier4.jp">Takagi, Isamu</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake_auto</buildtool_depend>
+
+  <build_depend>autoware_cmake</build_depend>
+  <build_depend>rosidl_default_generators</build_depend>
+
+  <depend>autoware_ad_api_msgs</depend>
+  <depend>autoware_auto_mapping_msgs</depend>
```

Comment:
I think we should remove this dependency in autoware_msgs.

```suggestion
```

Reply:
Currently, SetRoute service uses autoware_auto_mapping_msgs. It should be removed when moving autoware_auto_msgs to autoware_msgs.

```
std_msgs/Header header
geometry_msgs/Pose goal
autoware_auto_mapping_msgs/HADMapSegment[] segments
```

Reply:
Yes, I meant you should rewrite the message.

Reply:
I think @mitsudome-r can decide how to define this message.

Reply:
As a result of discussion with @mitsudome-r @kenji-miyake @yukkysaito , I think we now refer to autoware_auto_msg.

Reply:
I'm sorry but I didn't recognize we'll depend on `autoware_auto_msgs` here.
If we do so, we should discuss the approach with @xmfcx.

I suggest copying necessary messages from `autoware_auto_msgs` and depend on the new message package.

Reply:
Create the issue https://github.com/autowarefoundation/autoware_msgs/issues/34 for this.